l10n_it_account_stamp/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     'name': 'Italian Localization - Imposta di bollo',
-    'version': '12.0.1.0.3',
+    'version': '12.0.1.1.3',
```

Comment:
@primes2h ultima cosa: la versione dovrebbe essere `12.0.1.1.0`

Reply:
Hai ragione. Fatto!

Ho anche aggiunto un fix importante, il mese del decreto era errato (July al posto di June).
Ho fatto un commit separato in modo che sia possibile fare cherry-picking dalle altre versioni.
P.S.: come workaround l'avevo già corretto nella traduzione.

Reply:
In questo caso la versione andrebbe ulteriormente creata oppure includi https://github.com/OCA/l10n-italy/pull/891/commits/c2c3d893625d6fa9c1edc3fa679a467aea711848 nel commit precedente.

Reply:
> In questo caso la versione andrebbe ulteriormente creata oppure includi [c2c3d89](https://github.com/OCA/l10n-italy/commit/c2c3d893625d6fa9c1edc3fa679a467aea711848) nel commit precedente.

Eh... hai ragione, me ne sono dimenticato.
Ho incrementato la versione a ```12.0.1.1.1```, preferisco tenere il commit separato in questo caso.

Grazie!